projects/core/src/lib/mask.ts

```diff
@@ -12,6 +12,21 @@ import {
     maskitoTransform,
 } from './utils';
 
+function isRedo({ctrlKey, metaKey, shiftKey, code}: KeyboardEvent): boolean {
+    return (
+        (ctrlKey && code === 'KeyY') || // Windows
+        (ctrlKey && shiftKey && code === 'KeyZ') || // Windows & Android
+        (metaKey && shiftKey && code === 'KeyZ') // macOS & iOS
+    );
+}
+
+function isUndo({ctrlKey, metaKey, code}: KeyboardEvent): boolean {
```

Comment:
Thanks for your contribution!

Could you move both new utils to `projects/core/src/lib/utils/dom/history-events.ts`, please ?